docs/runbooks/HPPNotReady.md

```diff
@@ -1,32 +1,48 @@
+<!-- Edited by Jiří Herrmann, 3 Nov 2022 -->
+
 # HPPNotReady
 
 ## Meaning
 
-The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
-The HPPNotReady alert is indicative of a HPP installation being in a degraded state;  
+The `hostpath-provisioner` (HPP) is used to dynamically provision hostPath volumes to provide storage for persistent volume claims (PVCs).  
+The HPPNotReady alert indicates that an HPP installation is in a degraded state:
```

Comment:
Reverse the order of these paragraphs so that the meaning of the alert appears before the sentence about what the HPP does.

Also, let's change this to "This alert fires when an HPP installation..." for consistency.